src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -339,6 +339,7 @@ protected function describeContainerDefinition(Definition $definition, array $op
      */
     protected function describeContainerAlias(Alias $alias, array $options = array())
     {
+        $options['output']->setDecorated(false);
```

Comment:
This is a UI change for the console. Not sure its intended but i wouldnt change it or just use `writeln/block()` then.